### PR TITLE
Feature/add rgb hex validator

### PR DESCRIPTION
**Pull request by louim**
Thought that might be useful! 😉

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2014-2015, Mirego
+Copyright (c) 2014-2017, Mirego
```
Comment (remiprev) [inline LICENSE.md]: We’ll bump that when we’ll make a new release. No need to change it 😄

File: lib/louche.rb [hunk 2]
```diff
@@ -10,6 +10,7 @@
 require 'louche/validators/phone_number_validator'
 require 'louche/validators/postal_code_validator'
 require 'louche/validators/url_validator'
+require 'louche/validators/rgb_hex_validator'
```
Comment (remiprev) [inline lib/louche.rb]: These lines should be sorted in alphabetical order.

File: README.md [hunk 3]
```diff
@@ -163,6 +163,33 @@ User.new(tags: ['food', '', 'code']).valid? # => false
...
+
+#### Options
+
+| Option           | Description
+|------------------|-----------------------------------------------------
+| `:regex`         | The regex used to validate the code (default: `/\A#([a-f0-9]{6}|[a-f0-9]{3})\z/i`)
+| `:cleanup_regex` | The regex used to validate clean the input before validating/saving it (default: `/[^#a-f0-9]/i`)
+| `:message`       | The ActiveRecord message added to the record errors (default: `:invalid_rgb_hex`)
+
+
```
Comment (remiprev) [inline README.md]: Blank line cop! 👮

**Comment by remiprev** [comment]
I’ll update `.travis.yml` to remove Ruby 1.9.3 support…

**Comment by louim** [comment]
Done! I think we should also test with ActiveModel 5, I can do another PR for that.

**Comment by remiprev** [comment]
> Done! I think we should also test with ActiveModel 5, I can do another PR for that.

Already done in #8! 💥

**Comment by remiprev** [comment]
Can you rebase from `master`?

**Comment by louim** [comment]
Done and ✅

**Comment by remiprev** [comment]
Awesome! Thank you! 👍

**Pull request merged**